docs/INSTALL.md

```diff
@@ -314,43 +324,50 @@ Ensure that you have at least 30GB of free space.
 
 Install [Android Studio](https://developer.android.com/studio).
 
+Run Android Studio and open the project in `android/` directory!
+This is important, otherwise the following menus won't be visible.
+
 Install Android SDK and NDK:
 
-- Run the Android Studio.
 - Open "SDK Manager" (under "More Actions" in a welcome screen or a three-dot menu in a list of recent projects screen or "Tools" top menu item in an open project).
-- Select "Android 13.0 (T) / API Level 33" SDK.
+- Select "Android 13.0 ("Tiramisu") / API Level 33" SDK.
 - Switch to "SDK Tools" tab.
 - Check "Show Package Details" checkbox.
 - Select "NDK (Side by side)" version **26.0.10792818**.
 - Select "CMake" version **3.22.1**.
 - Click "Apply" and wait for downloads and installation to finish.
 - In the left pane menu select "Appearance & Behavior > System Settings > Memory Settings".
-- Set "IDE max heap size" to 2048Mb or more (otherwise the Studio might get stuck on "Updating indexes" when opening the project).
+- Set "IDE max heap size" to 2048MB or more (otherwise the Studio might get stuck on "Updating indexes" when opening the project).
 
 Configure the repository with Android SDK and NDK paths. You can do it either by setting a global environment variable pointing at your Android SDK:
+
 ```
 ANDROID_HOME=<here is the absolute path to the root folder of your Android SDK installation>
 ```
 
-or by running the following script, that creates `android/local.properties` file with the line `sdk.dir=<path to your Android SDK>` in it: 
+or by running the following script, that creates `android/local.properties` file with the line `sdk.dir=<path to your Android SDK>` in it:
 
 _Linux:_
+
 ```bash
 ./tools/android/set_up_android.py --sdk $HOME/Android/Sdk
 ```
 
 _macOS:_
+
 ```bash
 ./tools/android/set_up_android.py --sdk $HOME/Library/Android/Sdk
 ```
 
 _Windows 10:_ no action needed, should work out of the box.
 
-In Android Studio open the project in `android/` directory.
+### Create a device
 
-Setup a virtual device to use [emulator](https://developer.android.com/studio/run/emulator) ("Tools > Device Manager") or [use a hardware device for debugging](https://developer.android.com/studio/run/device). If using an emulator, make sure to choose a system image with API Level 31 and ABI _x86_64_.
+Setup a virtual device to use [emulator](https://developer.android.com/studio/run/emulator) ("Tools > Device Manager") or [use a hardware device for debugging](https://developer.android.com/studio/run/device).
+If using an emulator, make sure to choose a system image with API Level 31 and ABI _x86_64_.
```

Comment:
...The same api level for the SDK you installed?

Use ABI x86_64 for Intel-based processors and arm64-v8a for ARM-based processors (e.g. M1/M2 Macs)

Reply:
I did exactly what it told me to do. I will adjust it for M processors